### DISCUSSION TITLE
Instance with name

### DISCUSSION THREAD

**Discussion by TheDarkSword** [Minestom/Minestom#1732] [General]
Instances have only an UUID, there is a way to add a name like Bukkit Worlds?

**Comment by Tofaa2** [comment]
No, but you  can extend the instance container and add a name parameter. After that if you want it per folder simply set the source directory to the folder of your desire (im guessing server path + the world name). See the AnvilLoader class

**Reply by TheDarkSword** [reply]
Oh, thank you

**Comment by FluxCapacitor2** [comment]
Another potential solution is to use a [Tag](https://wiki.minestom.net/feature/tags) on each instance using [this method](https://javadoc.minestom.net/net/minestom/server/tag/Taggable.html#setTag(net.minestom.server.tag.Tag,T)), which might be easier than extending every instance type.
